src/math/lp/lp_core_solver_base.h

```diff
@@ -594,17 +595,16 @@ class lp_core_solver_base {
         }
         return true;
     }
-    unsigned & iters_with_no_cost_growing() {
+    unsigned& iters_with_no_cost_growing() {
         return m_iters_with_no_cost_growing;
     }
 
-    const unsigned & iters_with_no_cost_growing() const {
+    const unsigned& iters_with_no_cost_growing() const {
         return m_iters_with_no_cost_growing;
     }
 
-    
     unsigned get_base_column_in_row(unsigned row_index) const {
         return m_basis[row_index];
     }
 };
```

Comment:
I end up indenting everything under a namespace {} scope. 
The lp directory contains several files where the content under a namespace isn't indented. 
I opted so far to not "fix" this because it makes no functional change. Other code uses uniform indentation with {} scopes.